react-app/constants.js

```diff
@@ -1,18 +1,9 @@
-export const basicFeatures = ['View a list of folders and files', 'Navigate through folders', 'Download files', 'Preview files', 'Search files by name', 'Select files and/or folders'];
+export const basicDescriptionInstructions = 'Use this component to upload and organize files that students can access directly in the unit.';
+export const advancedDescriptionInstructions = `${basicDescriptionInstructions} Any files you upload will be added to the "File and uploads" section. Extracting files from the "unpublished" folder leaves them visible to students, returning them to the folder hides them.`;
```

Comment:
In the latest versions of the platform, it is not called "Files & Uploads" but only "Files". We could add those two.

```suggestion
export const advancedDescriptionInstructions = `${basicDescriptionInstructions} Any files you upload will be added to the "File & Uploads" or "Files" section. Extracting files from the "Unpublished" folder leaves them visible to students, returning them to the folder hides them.`;
```

Reply:
Done!
![edit-view-test](https://github.com/eduNEXT/xblock-filesmanager/assets/134975835/9618efe5-6500-4af0-b804-2185a2fc9292)